src/onegov/org/forms/settings.py

```diff
@@ -1161,6 +1163,14 @@ class NewsletterSettingsForm(Form):
         },
     )
 
+    notify_on_unsubscription = ChosenSelectMultipleEmailField(
+        label=_('Notify on newsletter unsubscription'),
+        description=_('Send an email notification when a user unsubscribes '
```

Comment:
I would write "send an email notification to the following users when a recipient unsubscribes"